kvmagent/kvmagent/plugins/bmv2_gateway_agent/volume/thirdparty_ceph.py

```
@@ -128,3 +138,25 @@ def get_all_access_path(self):
     def get_targets_by_access_path_id(self, access_path_id):
         return RbdDeviceOperator(self.volume_obj.monIp, self.volume_obj.token,
                                  self.volume_obj.tpTimeout).get_targets_by_access_path_id(access_path_id)
+
+    def build_iscsi_uri(self, gateway_ip):
```

Ruff (0.14.3)

Unused method argument: `gateway_ip`

(ARG002)